app/notification.go

```diff
@@ -17,11 +17,6 @@ import (
 	"github.com/mattermost/mattermost-server/utils/markdown"
 )
 
-const (
-	THREAD_ANY  = "any"
-	THREAD_ROOT = "root"
-)
-
 func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *model.Channel, sender *model.User, parentPostList *model.PostList) ([]string, error) {
```

Comment:
this function feels way too big. Not asking for a change, but if you like we can talk about changing it :)